app/src/app/components/specmath-navbar/specmath-navbar.component.html

```diff
@@ -0,0 +1,17 @@
+<!-- Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+<div class="nav-bar">
+  <img src="../assets//img/specmath logo.svg" class="nav-logo" />
```

Comment:
When we build angular must encode this image url but having a space in the name is not standard and there is an extra '/'

Reply:
Totally missed the extra space! I'll add an underscore and remove the extra '/'